modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-http-read-only/src/main/java/org/nuxeo/elasticsearch/http/readonly/Main.java

```diff
@@ -101,37 +100,58 @@ public String searchWithPayload(@PathParam("indices") String indices, @Context U
     @Produces(MediaType.APPLICATION_JSON)
     public String searchWithPost(@PathParam("indices") String indices, @Context UriInfo uriInf, String payload)
             throws IOException, JSONException {
-        return doSearchWithPayload(indices, "_all", uriInf.getRequestUri().getRawQuery(), payload);
+        return doSearchWithPayload(indices, uriInf.getRequestUri().getRawQuery(), payload);
     }
 
+    /**
+     * @deprecated since 11.4, types have been removed since Elasticsearch 7.x, use
+     *             {{@link #searchWithPayload(String, UriInfo, MultivaluedMap)} instead
+     */
     @GET
     @Path("{indices}/{types}/_search")
     @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
     @Produces(MediaType.APPLICATION_JSON)
+    @Deprecated
```

Comment:
don't we want the `since` in the annotation?

Reply:
Indeed, forRemoval added too.